vNext/extensions/applicationinsights-dependencies-js/src/ajax.ts

```diff
@@ -519,6 +523,13 @@ export class AjaxMonitor implements ITelemetryPlugin, IDependenciesPlugin, IInst
                         xhr.ajaxData.requestHeaders[RequestHeaders.requestContextHeader] = RequestHeaders.requestContextAppIdFormat + appId;
                     }
                 }
+                if (this._config.distributedTracingMode === DistributedTracingModes.AI_AND_W3C || this._config.distributedTracingMode === DistributedTracingModes.W3C) {
```

Comment:
Dupe of L499?

Reply:
this line if for xhr request, L499 is for fetch request